### ISSUE TITLE
APIError(code=-1013)

### ISSUE THREAD

**Issue by mauribuho** [ccxt/binance-trade-bot#442]
I keep getting this errors:
APIError(code=-1013: Invalidad Quantity
APIError(code=-1013: Filter failure: MIN_NOTIONAL

**Comment by kuklofon** [comment]
same error:
 2021-11-13 18:25:20,725 - crypto_trading_logger - INFO - APIError(code=-1013): Invalid quantity.

I tried with docker-compose up

**Comment by mauribuho** [comment]
> same error: 2021-11-13 18:25:20,725 - crypto_trading_logger - INFO - APIError(code=-1013): Invalid quantity.
> 
> I tried with docker-compose up

**Comment by mauribuho** [comment]
> same error: 2021-11-13 18:25:20,725 - crypto_trading_logger - INFO - APIError(code=-1013): Invalid quantity.
> 
> I tried with docker-compose up

Since i'm ussing the last version i have not been having that problem anymore, you downloaded the last files from 2 days ago?

**Comment by PetoFeFe** [comment]
I am having same problem, i just downloaded it

> 2021-11-15 18:15:11,591 - crypto_trading_logger - INFO - Chosen strategy: default
2021-11-15 18:15:11,591 - crypto_trading_logger - INFO - Creating database schema if it doesn't already exist
2021-11-15 18:15:42,202 - crypto_trading_logger - INFO - Will be jumping from [EOS] to BAT EOSUSDT
2021-11-15 18:15:42,214 - crypto_trading_logger - INFO - Skipping sell
2021-11-15 18:15:43,589 - crypto_trading_logger - INFO - BUY QTY 0.0 of <BAT>
2021-11-15 18:15:43,878 - crypto_trading_logger - INFO - APIError(code=-1013): Invalid quantity.
2021-11-15 18:15:45,677 - crypto_trading_logger - INFO - APIError(code=-1013): Invalid quantity.
2021-11-15 18:15:46,998 - crypto_trading_logger - INFO - APIError(code=-1013): Invalid quantity.
2021-11-15 18:15:48,280 - crypto_trading_logger - INFO - APIError(code=-1013): Invalid quantity.
2021-11-15 18:15:49,583 - crypto_trading_logger - INFO - APIError(code=-1013): Invalid quantity.

>

**Comment by MasaiasuOse** [comment]
Is your trade amount more than $10? It is the minimum trading volume of Binance

**Comment by PetoFeFe** [comment]
Yeah i have over 100€ balance, i think i fixed id when a sold coins for USDT, then it started working

**Comment by MasaiasuOse** [comment]
Probably you didn't set the user.cfg correctly and it tried to use USDT to start but didn't have enough

**Comment by zana31** [comment]
from heroku :

2021-11-16T22:39:52.908610+00:00 app[worker.1]: 2021-11-16 22:39:52,908 - crypto_trading_logger - INFO - APIError(code=-1013): Invalid quantity.

**Comment by kooltux** [comment]
Got the same while adding BTT or DOCK in supported coins. When the new crypto BTT was chosen, I got the error 1013  I'm not sure how minimum values are evaluated on BTB side (sorry, I had no time to dig into the code). But globally, before sending the order, BTB should be aware of the minimum value before submitting an order. That may be for a future enhancement?

But in any case, and not only for this specific error 1013: if the server sends an error, is it possible to "adjust" the strategy? (=drop the idea to put an order and swtich to the next). ATM, it seems that BTB retries infinitely to post the order (and we get error 1013 each time.... very annoying because you get notified for nothing and at the end, everything is blocked - you're good to restart the bot).

If we take a higher view, this is also a question regarding the strategy: if the bot can't push an order, what is the impact on the results? I hope this impact is minor, but I'm not so sure :) anybody to confirm?
